packages/api/amplify_api_dart/lib/src/graphql/factories/graphql_request_factory.dart

```diff
@@ -114,6 +114,7 @@ class GraphQLRequestFactory {
     ModelSchema schema,
     GraphQLRequestOperation operation,
     ModelIdentifier? modelIdentifier,
+    bool hasSubFilter,
```

Comment:
Is this needed? You should be able to add the variables without this. And it's kinda messy.